### ISSUE TITLE
enhancement: commands

### ISSUE THREAD

**Issue by TanguyOrtegat** [maxux/toogoodtogo-watcher#4]
add commands to the bot

like:
/refresh: gives you updated list 
/buy quantity storeid : to buy a basket
...

**Comment by maxux** [comment]
Since payment is not implemented yet, it won't work.
The bot try to get accurate refresh based on current time and random timewait to reduce ban risk. Changing code to allow user command just for that is too much for me right now.

The code is not able to handle user input and focus the logic on fetching tgtg update. This will maybe change later :)

**Comment by TanguyOrtegat** [comment]
Yes i do understand. Too bad I am such a noob at python am more javascript or c#.
In the future I ll try to look at it

**Comment by TanguyOrtegat** [comment]
I did look at the code you made. Did make some small fixes and managed to make basket work. The only thing now is how to make the payment work ...

**Comment by eimarfandino** [comment]
hey guys, so how does the checkout works? With the code now in master, i do a checkout and i see it all working and the item gets reserved, but then, how to proceed?

**Comment by maxux** [comment]
I'm amazed to see that checkout still works. I didn't touched that for long time and didn't had any time to look back at it.
I didn't used the app recently, I don't really have time to look at this :/

**Comment by eimarfandino** [comment]
nice work! for the checkout to work you need to change the api version from 4 to 6. I was wondering, once the item is reserved, how do you manage to open the basket?

**Comment by swiezaczek** [comment]
> I did look at the code you made. Did make some small fixes and managed to make basket work. The only thing now is how to make the payment work ...

Any updates? Did you manage to get basked reserved with all the api updates? If so, can you share it? @TanguyOrtegat
